ui-tests-protractor/e2e/connections/connections.steps.ts

```diff
@@ -110,6 +111,13 @@ class ConnectionSteps {
     const connectionsListComp = new ConnectionsListComponent();
     return connectionsListComp.checkAllKebabElementsAreDisplayed(true, actions);
   }
+
+  @then(/^connection "([^"]*)" is validated$/)
+  async connectionValidated (connectionName: string): P<any> {
+    //there will be alert-success class only if validation is successful
+    return await expect(element(by.className("alert-success")).isPresent(),
+        `Connection ${connectionName} was not successfully validated.`).to.eventually.be.true;
+  }
 }
```

Comment:
Maybe this could be re-used?

https://github.com/syndesisio/syndesis-qe/blob/9495bd9a2534cc06a10b5d486613f9572671c21b/ui-tests-protractor/e2e/common/common.steps.ts#L191-L195